global-test.properties

```diff
@@ -9,16 +9,20 @@ xtf.record.always=false
 # Avoid starting multiple builds at once as it might cause that builds starts to hang
 xtf.junit.prebuilder.synchronized=true
 
-# Bootable jar
+# Bootable JAR OpenJDK base image
 intersmash.bootable.jar.image=registry.access.redhat.com/ubi8/openjdk-17
 
-# WildFly community operator settings
+# WildFly operator settings
 intersmash.wildfly.operators.catalog_source=community-operators-wildfly-operator
 intersmash.wildfly.operators.index_image=quay.io/operatorhubio/catalog:latest
 intersmash.wildfly.operators.package_manifest=wildfly
 intersmash.wildfly.operators.channel=alpha
 
-# Kafka community operator settings
+# Wildfly images
+intersmash.wildfly.image=quay.io/wildfly/wildfly-s2i-jdk17
```

Comment:
@fabiobrz are these temporary images? just asking....

Reply:
Not really. Intersmash is community first. Those are the community projects images that we'll have in the global-test.properties.